WordPress/Classes/ViewRelated/Post/PostListViewController.m

```diff
@@ -0,0 +1,1319 @@
+                         if (err) {
+                             DDLogError(@"%@", err);
+                         }
+                         if (!post) {
+                             return;
+                         }
+
+                         // If the post was restored, see if it appears in the current filter.
+                         // If not, prompt the user to let it know under which filter it appears.
+                         PostListFilter *filter = [self filterThatDisplaysPostsWithStatus:post.status];
+                         if ([filter isEqual:[self currentPostListFilter]]) {
+                             return;
+                         }
+                         [self promptThatPostRestoredToFilter:filter];
+                     }
+                     failure:^(NSError *error) {
+                         if([error code] == 403) {
+                             [self promptForPasswordWithMessage:nil];
+                         } else {
+                             [WPError showXMLRPCErrorAlert:error];
+                         }
+                         [self.recentlyTrashedPostIDs addObject:postID];
+                     }];
+}
+
+- (void)promptThatPostRestoredToFilter:(PostListFilter *)filter
+{
+    NSString *message = NSLocalizedString(@"Post Restored to Drafts", @"Prompts the user that a restored post was moved to the drafts list.");
+    switch (filter.filterType) {
+        case PostListStatusFilterPublished:
+            message = NSLocalizedString(@"Post Restored to Published", @"Prompts the user that a restored post was moved to the published list.");
+            break;
+        case PostListStatusFilterScheduled:
+            message = NSLocalizedString(@"Post Restored to Scheduled", @"Prompts the user that a restored post was moved to the scheduled list.");
+            break;
+        default:
+            break;
+    }
+    UIAlertView *alertView = [[UIAlertView alloc] initWithTitle:nil
+                                                        message:message
+                                                       delegate:nil
+                                              cancelButtonTitle:NSLocalizedString(@"OK", @"Title of an OK button. Pressing the button acknowledges and dismisses a prompt.")
+                                              otherButtonTitles:nil, nil];
+    [alertView show];
+}
+
+- (void)viewStatsForPost:(AbstractPost *)apost
+{
+    // Check the blog
+    Blog *blog = apost.blog;
+    if (!blog.isWPcom) {
+        // Needs Jetpack.
+        return;
+    }
+
+    // Push the Stats Post Details ViewController
+    NSString *identifier = NSStringFromClass([StatsPostDetailsTableViewController class]);
+    BlogService *service = [[BlogService alloc] initWithManagedObjectContext:[[ContextManager sharedInstance] mainContext]];
+    UIStoryboard *statsStoryboard   = [UIStoryboard storyboardWithName:StatsStoryboardName bundle:nil];
```

Comment:
Related to the crash @SergioEstevao mentioned, since the storyboard is from a pod, `bundle` should probably not be `nil` here.